website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -55,42 +55,42 @@ provider "kubernetes" {
-* `service_subnet` - The range of assignable IP addresses for services running in the Kubernetes cluster.
-* `ipv4_address` - The public IPv4 address of the Kubernetes master node.
-* `endpoint` - The base URL of the API server on the Kubernetes master node.
-* `status` -  A string indicating the current status of the cluster. Potential values include running, provisioning, and errored.
-* `created_at` - The date and time when the Kubernetes cluster was created.
-* `updated_at` - The date and time when the Kubernetes cluster was last updated.
-* `kube_config.0` - A representation of the Kubernetes cluster's kubeconfig with the following attributes:
+- `id` - A unique ID that can be used to identify and reference a Kubernetes cluster.
+- `cluster_subnet` - The range of IP addresses in the overlay network of the Kubernetes cluster.
+- `service_subnet` - The range of assignable IP addresses for services running in the Kubernetes cluster.
+- `ipv4_address` - The public IPv4 address of the Kubernetes master node.
+- `endpoint` - The base URL of the API server on the Kubernetes master node.
+- `status` - A string indicating the current status of the cluster. Potential values include running, provisioning, and errored.
+- `created_at` - The date and time when the Kubernetes cluster was created.
+- `updated_at` - The date and time when the Kubernetes cluster was last updated.
+- `kube_config.0` - A representation of the Kubernetes cluster's kubeconfig with the following attributes:
   - `raw_config` - The full contents of the Kubernetes cluster's kubeconfig file.
   - `host` - The URL of the API server on the Kubernetes master node.
   - `client_key` - The base64 encoded private key used by clients to access the cluster.
   - `client_certificate` - The base64 encoded public certificate used by clients to access the cluster.
   - `cluster_ca_certificate` - The base64 encoded public certificate for the cluster's certificate authority.
-* `node_pool` - In addition to the arguments provided, these additional attributes about the cluster's default node pool are exported:
-  - `id` -  A unique ID that can be used to identify and reference the node pool.
+- `node_pool` - In addition to the arguments provided, these additional attributes about the cluster's default node pool are exported:
+  - `id` - A unique ID that can be used to identify and reference the node pool.
   - `nodes` - A list of nodes in the pool. Each node exports the following attributes:
-     + `id` -  A unique ID that can be used to identify and reference the node.
-     + `name` - The auto-generated name for the node.
-     + `status` -  A string indicating the current status of the individual node.
-     + `created_at` - The date and time when the node was created.
-     + `updated_at` - The date and time when the node was last updated.
+    - `id` - A unique ID that can be used to identify and reference the node.
```

Comment:
Looks like something is slightly off with the formatting here. I think they need to be indent one more level.

Compare this:

![Selection_999(1306)](https://user-images.githubusercontent.com/46943/61660910-f3fd9700-ac98-11e9-827e-2879fe6e4d86.png)

To the live version: https://www.terraform.io/docs/providers/do/r/kubernetes_cluster.html#nodes

You can test your changes by running `make website` and navigating to `http://127.0.0.1:4567/docs/providers/do/r/kubernetes_cluster.html`

Reply:
Ah, sorry about that!